tao_compiler/mlir/disc/transforms/disc_gpu_kernel_to_blob.cc

```diff
@@ -305,9 +305,24 @@ class GpuKernelToBlobPass
 
     xla::HloModuleConfig config;
     xla::DebugOptions options = xla::GetDebugOptionsFromFlags();
-    // Make sure we use full precision division operations.
-    // We need this to pass the ut for `tf.FloorDiv`.
-    (*options.mutable_xla_backend_extra_options())["-nvptx-prec-divf32"] = "2";
```

Comment:
Is this is ok now? i remember there was a UT failure if we open this flag by default.

Reply:
You remember right that this flag should be set to "2" for the UT of FloorDiv. It is moved to line-323, which is still the default setting.